### DISCUSSION TITLE
List of all content_types

### DISCUSSION THREAD

**Discussion by AsMaNick** [eternnoir/pyTelegramBotAPI#1880] [Q&A] [answered]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?
4.8.0
2. What OS are you using?
Both Linux/Windows 10
3. What version of python are you using?
Python 3.9.0

Where is it possible to see the list of all available `content_types` for the `message_handler` decorator? There's some list in [https://github.com/eternnoir/pyTelegramBotAPI#types](https://github.com/eternnoir/pyTelegramBotAPI#types), but it seems to be incomplete because it doesn't contain at least `animation` type.

Also it will be convenient to have a simple option to specify all content_types, not only by specifying a big list of all possibile strings.

**Comment by Badiboy** [comment, marked as answer]
https://core.telegram.org/bots/api#message

Here.

**Comment by Badiboy** [comment]
Starting from "text" and down, except "entities".
